### PR TITLE
[README] Delete duplicate layer description and Fix typo errors.

### PR DESCRIPTION
**Pull request by baek2sm**
Fix some minor errors in README.

- Delete pooling2D layer description because it was duplicated twice.
- Fix some typo errors.

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.net>

### CODE REVIEW

**Comment by baek2sm** [comment]
I will check and correct the CI error(about signed-off).

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM! Nice Catch👍

File: README.md [hunk 1]
```diff
@@ -143,13 +142,13 @@ NNTrainer provides
  |:-------:|:---:|:---|
  | tanh | tanh function | set as layer property |
  | sigmoid | sigmoid function | set as layer property |
- | relu | relu function | set as layer propery |
- | softmax | softmax function | set as layer propery |
+ | relu | relu function | set as layer property |
```
Comment (jihochu) [inline README.md:145]: Nice catch!!

**Review by jihochu — APPROVED** [review]
Docs are looking up. :+1:

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**